packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -58,6 +58,7 @@ export const enableUseRefAccessWarning = false;
 export const enableRecursiveCommitTraversal = false;
 export const disableSchedulerTimeoutInWorkLoop = false;
 export const enableLazyContextPropagation = false;
+export const enableSyncDefaultUpdates = false;
```

Comment:
I think it should be `true` everywhere except ReactFeatureFlags.www-dynamic, right?

Reply:
I feel like it mostly only exists so we get the extra test coverage for free. Since internally we plan to use the root override prop.

Reply:
I haven't given it much thought, just pushed to get the tests running. That seems reasonable to me.

Reply:
Sorry, missed your second comment. Don't we want to assume it's off as much as possible? I.e. default to the future behavior?

Reply:
Yeah which means we almost don't even need the flag. It only exists to make it easier for us to toggle the behavior globally in our test suite without having to add a dynamic override to every `createRoot` call.

Reply:
Right, I think this is doing what you want which is to use the new behavior everywhere except tests. This flag turns on the old behavior, so shouldn't it be false everywhere except the file that sets it to VARIANT?

Reply:
>  This flag turns on the old behavior

Is that true? It's called `enableSyncDefaultUpdates`. Maybe the name is inverted and you meant to call it `enableConcurrentDefaultUpdates`?

But looking at the implementation, it does look like when the flag is `true`, the updates are sync by default.

`createRoot` should create a sync-by-default root in 18. So `enableSyncDefaultUpdates` should be true.

What am I missing?